x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -12,10 +12,18 @@
 import org.elasticsearch.features.NodeFeature;
 
 import java.util.Map;
+import java.util.Set;
 
 public class EsqlFeatures implements FeatureSpecification {
+    private static final NodeFeature MV = new NodeFeature("esql.mv");
```

Comment:
very subjective nit: I'd prefer to have a more descriptive name here, like `LUCENE_MV_WARNING` and `"esql.lucene_mv_warning"`

Reply:
This was just loading multivalue fields at all.

Reply:
Oh, I see, never mind, I thought it was some more subtle change